### PR TITLE
Entitlements endpoint fixes

### PR DESCRIPTION
**Pull request by vinibackbase**
Evidence
Screen Shot 2020-03-17 at 14 29 13

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
             <dependency>
                 <groupId>com.backbase.dbs</groupId>
                 <artifactId>banking-services-bom</artifactId>
-                <version>2.17.3</version>
+                <version>2.18.0-cr.37</version>
```
Comment (richard-blank) [inline pom.xml]: you only can use versions that have been released to public repo

Comment (vinibackbase) [reply pom.xml]: Hi Richard, the teams need to use this particular version in order of them to ingest data with the current in development version (soon to be released) 2.18.0 dbs version to validate their changes.

Comment (richard-blank) [reply pom.xml]: That's only possible with a release that has been built internally. I have added this local build as `pre-release` (https://github.com/Backbase/bb-fuel/releases/tag/bb-fuel-2.5.14-alpha) that can be used in the pipelines by enabling `PRERELASE` and latest version

File: src/main/java/com/backbase/ct/bbfuel/client/accessgroup/AccessGroupIntegrationRestClient.java [hunk 2]
```diff
@@ -7,12 +7,14 @@
...
 import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.config.functions.FunctionsGetResponseBody;
-import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.datagroups.DataGroupPostRequestBody;
+import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.datagroups.IntegrationDataGroupCreate;
 import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.functiongroups.FunctionGroupPostRequestBody;
 import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.users.permissions.IntegrationAssignUserPermissions;
 import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.users.permissions.IntegrationFunctionGroupDataGroup;
 import io.restassured.http.ContentType;
 import io.restassured.response.Response;
+
+import java.util.ArrayList;
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/client/accessgroup/AccessGroupIntegrationRestClient.java]: unused import

File: src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java [hunk 3]
```diff
@@ -2,21 +2,22 @@
...
-import com.backbase.presentation.limit.rest.spec.v2.limits.PeriodicLimitsPostRequestBody;
-import com.backbase.presentation.limit.rest.spec.v2.limits.TransactionalLimitsPostRequestBody;
+import com.backbase.presentation.limit.rest.spec.v2.limits.CreateLimitRequestBody;
 import io.restassured.http.ContentType;
 import io.restassured.response.Response;
-import javax.annotation.PostConstruct;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
+import javax.annotation.PostConstruct;
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java]: unused import

File: src/main/java/com/backbase/ct/bbfuel/client/accessgroup/AccessGroupIntegrationRestClient.java (hunk 2 above)
Comment (vinibackbase) [reply src/main/java/com/backbase/ct/bbfuel/client/accessgroup/AccessGroupIntegrationRestClient.java]: Fixed!

File: pom.xml (hunk 1 above)
Comment (vinibackbase) [reply pom.xml]: Oh, okay, thanks for letting me know, switched back to 2.17.3

File: src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java [hunk 4]
```diff
@@ -26,17 +27,17 @@
     public void init() {
         setBaseUri(config.getPlatform().getGateway());
         setVersion(SERVICE_VERSION);
-        setInitialPath(LIMITS_PRESENTATION_SERVICE);
+        setInitialPath(LIMITS_PRESENTATION_SERVICE + "/" + CLIENT_API);
     }
 
-    public Response createPeriodicLimit(PeriodicLimitsPostRequestBody periodicLimitsPostRequestBody) {
+    public Response createPeriodicLimit(CreateLimitRequestBody periodicLimitsPostRequestBody) {
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java]: this method can be removed as it's not being used

File: src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java [hunk 5]
```diff
@@ -26,17 +27,17 @@
...
         return requestSpec()
             .contentType(ContentType.JSON)
             .body(periodicLimitsPostRequestBody)
             .post(getPath(ENDPOINT_PERIODIC));
     }
 
-    public Response createTransactionalLimit(TransactionalLimitsPostRequestBody transactionalLimitsPostRequestBody) {
+    public Response createTransactionalLimit(CreateLimitRequestBody transactionalLimitsPostRequestBody) {
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java]: this endpoint has been deprecated and will be removed in 2.18.0 so better use the new one now

Comment (richard-blank) [reply src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java]: we agreed to postpone for now (will do later)

**Pull request merged**